src/ui/server/controllers/insurance/policy/loss-payee-details/index.ts

```diff
@@ -58,13 +69,20 @@ export const get = (req: Request, res: Response) => {
     return res.redirect(PROBLEM_WITH_SERVICE);
   }
 
+  /**
+   * maps application
+   * maps nominated loss payee LOCATION to IS_LOCATED_IN_UK or IS_LOCATED_INTERNATIONALLY
+   */
```

Comment:
I don't think we need this comment, it's describing what `mapApplicationToFormFields` is doing